streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchQuery.java

```diff
@@ -99,6 +94,10 @@ public void setBatchSize(int batchSize) {
     }
 
     public void setScrollTimeout(String scrollTimeout) {
+        this.scrollTimeout = TimeValue.parseTimeValue(scrollTimeout, TimeValue.timeValueMillis(5 * 1000 * 60));
```

Comment:
Is there a default value being used for the scrollTimeout in the event that the user does not specify one?